### DISCUSSION TITLE
How to use multiple configs for different menus?

### DISCUSSION THREAD

**Discussion by stevensonmt** [davatorium/rofi#1665] [Q&A] [answered]
I can't quite figure out how to run rofi and point to different configs for different types of menus. So if I want a launcher use launcher.rasi or powermenu use powermenu.rasi and so on. Can someone point me to the right documentation?

**Comment by DaveDavenport** [comment, marked as answer]
The manpage (**rofi(1)**) should be the right starting point.

**Comment by stevensonmt** [comment]
Ah, so there is a -config filename option. I checked rofi --help but did not see this option there. Thanks for the tip.

**Reply by DaveDavenport** [reply]
It is only one `-` not two.
